src/generate-project/index.ts

```diff
@@ -27,6 +27,12 @@ const SUPPORTED_PROJECT_TEMPLATES: Array<ProjectTemplate> = [
   'angular',
 ];
 
+const COPY_FONTS_MAPPER: Record<ProjectTemplate, string> = {
```

Comment:
A mapper sounds like an object with methods to map something. Perhaps  rename to `..._MAP`.